src/java.desktop/macosx/classes/com/apple/laf/AquaTableHeaderBorder.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2014, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The first date is the date when the file was created, the second date is a date when it was updated last time. So you do not need to add a new one, just update the second.

Reply:
Updated the year